test/integration/retryable-writes/retryable_writes.spec.test.ts

```diff
@@ -1,207 +1,27 @@
-
-    const actual = result.value ?? result;
-    // Some of our result classes contain the optional 'acknowledged' property which is
-    // not part of the test expectations.
-    for (const property in expected) {
-      expect(actual).to.have.deep.property(property, expected[property]);
-    }
-  }
-
-  if (test.outcome.collection) {
-    const collectionResults = await ctx.collection.find({}).toArray();
-    expect(collectionResults).to.deep.equal(test.outcome.collection.data);
-  }
-}
-
-// Helper Functions
-
-/** Transforms the arguments from a test into actual arguments for our function calls */
-function generateArguments(test) {
-  const args = [];
-
-  if (test.operation.arguments) {
-    const options: Record<string, any> = {};
-    for (const arg of Object.keys(test.operation.arguments)) {
-      if (arg === 'requests') {
-        /** Transforms a request arg into a bulk write operation */
-        args.push(
-          test.operation.arguments[arg].map(({ name, arguments: args }) => ({ [name]: args }))
-        );
-      } else if (arg === 'upsert') {
-        options.upsert = test.operation.arguments[arg];
-      } else if (arg === 'returnDocument') {
-        options.returnDocument = test.operation.arguments[arg].toLowerCase();
-      } else {
-        args.push(test.operation.arguments[arg]);
-      }
-    }
-
-    if (Object.keys(options).length > 0) {
-      args.push(options);
-    }
-  }
-
-  return args;
-}
-
-/** Runs a command that turns off a fail point */
-async function turnOffFailPoint(client, name) {
-  return await client.db('admin').command({
-    configureFailPoint: name,
-    mode: 'off'
-  });
-}
+const clientBulkWriteTests = [
+  'client bulkWrite with one network error succeeds after retry',
+  'client bulkWrite with two network errors fails after retry',
+  'client bulkWrite with no multi: true operations succeeds after retryable top-level error',
+  'client bulkWrite with multi: true operations fails after retryable top-level error',
+  'client bulkWrite with no multi: true operations succeeds after retryable writeConcernError',
+  'client bulkWrite with multi: true operations fails after retryable writeConcernError',
+  'client bulkWrite with retryWrites: false does not retry',
+  'client.clientBulkWrite succeeds after retryable handshake network error',
+  'client.clientBulkWrite succeeds after retryable handshake server error (ShutdownInProgress)'
+];
 
 describe('Retryable Writes (unified)', function () {
-  runUnifiedSuite(loadSpecTests(path.join('retryable-writes', 'unified')));
+  runUnifiedSuite(loadSpecTests(path.join('retryable-writes', 'unified')), ({ description }) => {
+    return clientBulkWriteTests.includes(description)
+      ? `TODO(NODE-6257): implement client-level bulk write.`
+      : description ===
+        'RetryableWriteError label is not added based on writeConcernError in pre-4.4 mongos response'
+      ? 'TODO(NODE-5720)'
+      : false;
+  });
 });
```

Comment:
Looks great! @nbbeeken, this is what I mentioned I wanted to check for in our meeting.